### ISSUE TITLE
Troubleshoot: Login Auth

### ISSUE THREAD

**Issue by wollmancsp** [PhillipBen/myapplication#6]
As a Concordia engineer I would like to be able to login to the application. When the application is spun up, it says you are able to login using admin:admin and user:user as credentials for username:password. This is not the case. This ticket represents the work to troubleshoot the issue and remedy.

**Comment by PhillipBen** [comment]
Dustin has been able to initially get the admin/user base accounts running. I can second that. We are currently having differences in connecting with the MySQL database though. Dustin got his to login with the application-dev.yml via the following:

`  datasource:
    type: com.zaxxer.hikari.HikariDataSource
    url: jdbc:mysql://localhost:3306/myapplication?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&createDatabaseIfNotExist=true
    username: root
    password: toor
    hikari:
      poolName: Hikari
      auto-commit: false
      data-source-properties:
        cachePrepStmts: true
        prepStmtCacheSize: 250
        prepStmtCacheSqlLimit: 2048
        useServerPrepStmts: true`

I on the other hand, was not able to get it logged in even after matching database url info & user/pass. I tried adding to the pom file, as was discussed in online discussion boards, but it didn't change anything. 
My current solution was just to set the MySQL password to null. This 'works', but should eventually be fixed.

**Comment by PhillipBen** [comment]
Today I also worked on new user registration. I created an email account for the application, and put in the credentials into the application-dev.yml file. However, Gmail doesn't like 3rd party applications accessing the account, so I had to 2 factor verify it and create an application password that JHipster could use. Now it works, and users can authenticate themselves by putting in a password. JHipster will pass the registration email through the proxy email to the user, then when the user clicks the link they will be authenticated, allowing for a full login to the system.
